models/org_team.go

```diff
@@ -496,6 +496,12 @@ func AddTeamMember(team *organization.Team, userID int64) error {
 	}
 	defer committer.Close()
 
+	// check in transaction
+	isAlreadyMember, err = organization.IsTeamMember(ctx, team.OrgID, team.ID, userID)
+	if err != nil || isAlreadyMember {
```

Comment:
But `err` is `nil` when `isAlreadyMember` is true.

Reply:
it will return nil than, this should be fine